dashboard/config/initializers/active_model_serializers.rb

```diff
@@ -12,20 +12,4 @@ def attributes(obj)
 end
 class ActiveModel::ErrorsSerializer < HashSerializer; end
 
-# Patch to provide more informative missing-serializer error message
```

Comment:
+1 This same change is in my PR (with some context in the description): https://github.com/code-dot-org/code-dot-org/pull/8451